cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ListsTest.scala

```diff
@@ -46,6 +46,11 @@ class ListsTest extends DocumentingTest {
       """* <<cypher-lists-general,Lists in general>>
         |* <<cypher-list-comprehension,List comprehension>>
         |* <<cypher-pattern-comprehension,Pattern comprehension>>""")
+    note{
+      p(
+        """Information regarding operators such as list concatenation (`+`), element existence checking (`IN`) and access (`[]`) can be found <<query-operators-list, here>>.
+          |The behaviour of the `IN` and `[]` operators with respect to `null` is detailed <<cypher-working-with-null, here>>.""".stripMargin)
```

Comment:
In four places in the PR: Should be _behavior_ as we use AM-EN in the manuals.
I can change this and then merge. Shall I?

Reply:
Yes, please go ahead